semantics/small-steps-test/small-steps-test.cabal

```diff
@@ -33,6 +33,7 @@ library
                      , microlens-th
                      , mtl
                      , nothunks
+                     , prettyprinter
```

Comment:
How is this related to this PR?

Reply:
It seems the tests won't build without it. See Issue 2093, and the transcript of the build here
https://github.com/input-output-hk/cardano-ledger-specs/issues/2093